.github/workflows/appetize-validation.yml

```
@@ -0,0 +1,131 @@
+name: AppetizeValidation
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: 0 14 * * *
+
+jobs:
+  website:
+    name: Resolve website config
+    runs-on: ubuntu-latest
+    outputs:
+      config: ${{ steps.config.outputs.result }}
+    steps:
+      - name: 👀 Checkout repository
+        uses: actions/checkout@v2
+      
+      - name: 🚀 Use Node 16
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16.x
+
+      - name: 🔍 Find yarn cache
+        id: yarn-cache
+        run: echo "::set-output name=path::$(yarn cache dir)"
+      
+      - name: ♻️ Restore yarn cache
+        uses: actions/cache@v2
+        with:
+          path: ${{ steps.yarn-cache.outputs.path }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+
+      - name: 🧶 Install dependencies
+        working-directory: website
+        run: yarn install --ignore-scripts --frozen-lockfile
+
+      # This command might fail on other dependencies, we just want to peak in the configs
+      - name: 👷‍♀️ Build sdk config
+        working-directory: website
+        continue-on-error: true
+        run: yarn tsc src/client/configs/sdk.tsx > /dev/null
+
+      - name: 🕵️ Resolve sdk config
+        uses: actions/github-script@v4
+        id: config
+        with:
+          script: |
+            const config = require('./website/src/client/configs/sdk')
+            const versions = Object.keys(config.versions)
+            
+            core.info(`Resolved suppored website SDKs`)
+            for (const version of versions) {
+              core.info(`  - ${version} ${version === config.DEFAULT_SDK_VERSION ? '(default)' : ''}`)
+            }
+
+            return { versions, default: config.DEFAULT_SDK_VERSION }
+
+  appetize:
+    needs: website
+    name: Validate ${{ matrix.platform }} on ${{ matrix.appetize }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        # Use capitalized names for envvars
+        appetize: ['MAIN', 'EMBED']
+        platform: ['ANDROID', 'IOS']
+    steps:
+      - name: 🚀 Use Node 16
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16.x
+          
+      - name: 📋 Download Appetize information
+        # This step MUST only output `appVersionName`, the endpoint returns the public and private app keys
```

```suggestion
        # Redirect output because the endpoint returns the public and private app keys
```

maybe copy/paste typo re: appVersionName?